api/src/main/resources/org/openmrs/api/db/hibernate/OrderSet.hbm.xml

```diff
@@ -36,6 +36,9 @@
             </type>
         </property>
 
+        <many-to-one name="Concept" class="org.openmrs.Concept"
```

Comment:
@dkayiwa I hope this is the right name, looked through a couple of classes and hbm file

Reply:
Have you read any hibernate mapping files tutorials? If yes, can you share the links here?

Reply:
https://www.tutorialspoint.com/hibernate/index.htm and https://www.javatpoint.com/one-to-one-mapping-in-hibernate-by-many-to-one-example

Reply:
What did those tutorials say that the name attribute is supposed to represent?

Reply:
Refers to the property in the class

Reply:
So is that what your changes reflect?

Reply:
oh yea, looks like I wasn't paying attention